Automattic/wp-calypso
# Plans: Refactor `PlanFeaturesHeader` tests to `@testing-library/react`

#### Proposed Changes

This PR refactors the `PlanFeaturesHeader` tests to use `@testing-library/react`

We use snapshot testing for prices, because the markup is not accessible and would require a lot of `querySelector` queries to make things right, while a snapshot test ensures the number and order of prices and their markup is right. This isn't ideal, but it allows us to make a step towards improving the tests of that component.

#### Testing Instructions

Verify tests still pass: `yarn run test-client client/my-sites/plan-features/test/header.jsx`

#### Pre-merge Checklist






Related to #63409
